### PR TITLE
Support using custom docker image

### PR DESCRIPTION
**Pull request by astj**
was #22

Also I added `MackerelContainerAgentImage.Latest` and `MackerelContainerAgentImage.Plugins`. Users just want to use official plugins can use `Plugins` at docker image specification.

### CODE REVIEW

File: src/mackerel-container-agent-definition.ts [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import { AmazonLinuxGeneration } from "@aws-cdk/aws-ec2"
```
Comment (astj) [inline src/mackerel-container-agent-definition.ts]: Hmm, I forgot to remove this import in #63. (I wonder tslint didn't complain at that time 🤔 )

**Comment by aereal** [comment]
@astj Is this Pull Request ready to review?

**Comment by astj** [comment]
@aereal I believe it is. Can you review this?

File: src/mackerel-container-agent-definition.ts [hunk 2]
```diff
@@ -11,15 +10,21 @@ import {
   ServiceRole,
 } from "./types"
 
-export interface Props
-  extends Pick<
-    ContainerDefinitionProps,
-    Exclude<keyof ContainerDefinitionProps, "image">
-  > {
+export interface Props extends Omit<ContainerDefinitionProps, "image"> {
```
Comment (aereal) [inline src/mackerel-container-agent-definition.ts]: :+1:

File: src/mackerel-container-agent-definition.ts [hunk 3]
```diff
@@ -11,15 +10,21 @@ import {
...
   ignoreContainer?: string
   hostStatusOnStart?: MackerelHostStatus
+  image?: ContainerImage
+}
+
+export const MackerelContainerAgentImage = {
+  Latest: ContainerImage.fromRegistry(
+    "mackerel/mackerel-container-agent:latest"
+  ),
+  Plugins: ContainerImage.fromRegistry(
```
Comment (aereal) [inline src/mackerel-container-agent-definition.ts]: I think `plugins` lacks some contexts in constrast of `latest` that includes no additional plugins, however, this naming is derived from mackerel-container-agent so this matter is not of this library...

Comment (astj) [reply src/mackerel-container-agent-definition.ts]: Hmm.

For example `golang:windowsservercore` is actually "Windows version of `golang:latest`" and I think it's acceptable, therefore the naming `plugin` is not so bad...
ref: https://hub.docker.com/_/golang

**Comment by astj** [comment]
Thank you for review!

**Pull request merged**